pages/dmp_supporting_info/support_06_document.md

```diff
@@ -16,6 +16,15 @@ cessda:
 rdmkit:
 - name: Documentation and metadata
   url: https://rdmkit.elixir-europe.org/metadata_management
+OpenAIRE:
```

Comment:
This currently won't work. If we want to add the additional links in this way, we should add them to `_includes/more-information-tiles.html`. I think this only makes sense if we re-use them at different places. We can also discuss having a generic icon.
The editorial group should take a decision on this.